bin/lint

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+shellcheck \
```

Comment:
Should we add this dependency to the development container as well, so we can run it locally and don't need to rely on CI when editing scripts?

Reply:
Ah yes, good call. I'm sorry about the auto-merge. I'll fire up another PR to add this.

Reply:
I opened this: https://github.com/dependabot/dependabot-core/pull/3938